HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,562 @@
+                            "ReadRequirement": "IfImplemented"
+                        },
+                        "ServiceEnabled": {},
+                        "ServiceAddresses": {},
+                        "RemoteRoleMapping": {}
+                    }
+                },
+                "OAuth2": {
+                    "ReadRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OAuth2Service": {},
+                        "ServiceEnabled": {}
+                    }
+                },
+                "OutboundConnections": {
+                    "ReadRequirement": "Recommended"
+                },
+                "SupportedAccountTypes": {}
+            }
+        },
+        "Certificate": {
+            "ReadRequirement": "Mandatory",
+            "URIs": "/redfish/v1/Managers/{ManagerId}/NetworkProtocol/HTTPS/Certificates/{CertificateId}",
+            "PropertyRequirements": {
+                "CertificateString": {}
+            }
+        },
+        "CertificateService": {
+            "ReadRequirement": "Mandatory",
+            "PropertyRequirements": {
+                "CertificateLocations": {}
+            },
+            "ActionRequirements": {
+                "GenerateCSR": {},
+                "ReplaceCertificate": {}
+            }
+        },
+        "EthernetInterface": {
+            "ReadRequirement": "Mandatory",
+            "URIs": [
+                "/redfish/v1/Managers/{ManagerId}/EthernetInterfaces/{EthernetInterfaceId}"
+            ],
+            "PropertyRequirements": {
+                "MACAddress": {},
+                "SpeedMbps": {
+                    "ReadRequirement": "Supported"
+                },
+                "InterfaceEnabled": {},
+                "LinkStatus": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "DHCPv4": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "DHCPEnabled": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "DHCPv6": {
+                    "ReadRequirement": "Recommended",
+                    "WriteRequirement": "Recommended",
+                    "PropertyRequirements": {
+                        "OperatingMode": {
+                            "ReadRequirement": "IfImplemented"
+                        }
+                    }
+                },
+                "HostName": {
+                    "ReadRequirement": "Supported"
+                },
+                "FQDN": {
+                    "ReadRequirement": "Supported"
+                },
+                "NameServers": {},
+                "IPv4Addresses": {},
```

Comment:
Should there be property requirements like with inside IPv6Addresses? Should we also call out property requirements inside the static addresses properties?

Reply:
Everything inside of `IPv4Addresses` is mandatory, correct?  (I was shortcutting, but doesn't hurt to call them out)

Reply:
Nope, nothing in the schema marks them as required today.